modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/suggestions/spi/SXPBlueprintSuggestionsContributor.java

```diff
@@ -97,11 +97,13 @@ public SuggestionsContributorResults getSuggestionsContributorResults(
 			(Map<String, Object>)
 				suggestionsContributorConfiguration.getAttributes();
 
-		if ((attributes == null) || !attributes.containsKey("sxpBlueprintId")) {
+		if ((attributes == null) ||
+			!attributes.containsKey("sxpBlueprintExternalReferenceCode")) {
```

Comment:
I think this contributor should keep the possibility to rely also on ID because afaik there already are customers using the REST endpoint directly from their widget templates. Upgrade processes won't cover / help them. 

Otherwise this would be a breaking change. cc: @lipusz